src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -357,20 +355,6 @@ public WebDriver createWebDriver(TestCleaner cleaner, TestName testName, Elastic
         cleaner.addTask(new Statement() {
             @Override
             public void evaluate() {
-                switch (getBrowser()) {
-                    case "firefox":
-                    case "saucelabs-firefox":
-                    case "remote-webdriver-firefox":
-                        // https://github.com/mozilla/geckodriver/issues/1151
-                        // https://bugzilla.mozilla.org/show_bug.cgi?id=1264259
-                        // https://bugzilla.mozilla.org/show_bug.cgi?id=1434872
-                        d.navigate().to("about:mozilla");
-                        Alert alert = ExpectedConditions.alertIsPresent().apply(d);
-                        if (alert != null) {
-                            alert.accept();
-                            d.navigate().refresh();
-                        }
-                }
```

Comment:
This is far more serious than I thought. The fact that the `d.quit()` line below is not in a `finally` block means that any error in this workaround will leak a session, negatively affecting future tests.